scripts/rook.sh

```diff
@@ -123,6 +132,7 @@ function teardown_rook() {
 	kubectl delete -f "${ROOK_URL}/toolbox.yaml"
 	kubectl delete -f "${ROOK_URL}/cluster-test.yaml"
 	kubectl delete -f "${ROOK_URL}/operator.yaml"
+	kubectl delete -f "${ROOK_URL}/crds.yaml" --ignore-not-found
```

Comment:
```bash
$ kubectl delete -f "crds.yaml" --ignore-not-found
error: the path "crds.yaml" does not exist
```
This does seem to throw error, if the file is not found(not part of rook <= v1.4), better to have the rook version check here too?
```suggestion
        # If rook version is > 1.5 , delete applied CRDs.
        ROOK_MAJOR=$(rook_version 1)
        ROOK_MINOR=$(rook_version 2)
        if  [ "${ROOK_MAJOR}" -eq 1 ] && [ "${ROOK_MINOR}" -ge 5 ];
	  then
	      kubectl delete -f "${ROOK_URL}/crds.yaml"
	  fi
```

Reply:
That will, but wondering do we need to care about this error? for safer side we can add it though